docs/Apache.md

```diff
@@ -31,7 +35,22 @@ my installation, the uid is 48. Symbolic uids are also supported (e.g.,
 We add a new section to the gssproxy configuration.  To do this, copy the
 ```examples/80-httpd.conf``` file to ```/etc/gssproxy/80-httpd.conf```.  (If
 you are using a monolithic config file at ```/etc/gssproxy/gssproxy.conf```,
-make sure the HTTP stanza preceeds any ```allow_any_uid=yes``` sections.)
+make sure the HTTP stanza precedes any ```allow_any_uid=yes``` sections.)
+
+For the second goal, the proxy will require a keytab for the user principal (apache@REALM).  Again, the uid used here is 48, but it must match whatever httpd is running as.
```

Comment:
So while this is a valid choice, it is not required. It is as well possible to simply map the HTTP/fqdn principal to an "apache" user on the server for example.
So I think we should rephrase this bit something like "a principal that maps to the correct user on the server", then you can make an example using apache@REALM...

Reply:
"NFS Server"